ui/src/app/dashboard/views/DiscoveriesList/DiscoveriesList.test.tsx

```diff
@@ -135,4 +135,32 @@ describe("DiscoveriesList", () => {
       .simulate("click");
     expect(wrapper.find("[data-test='delete-discovery']").exists()).toBe(true);
   });
+
+  it("can delete a discovery", () => {
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/dashboard", key: "testKey" }]}
+        >
+          <DiscoveriesList />
+        </MemoryRouter>
+      </Provider>
+    );
+    // Open the action menu.
+    wrapper
+      .find("[data-test='row-menu'] button.p-contextual-menu__toggle")
+      .first()
+      .simulate("click");
+    // Click on the delete link.
+    wrapper
+      .find("button[data-test='delete-discovery-link']")
+      .first()
+      .simulate("click");
+    // Click on the confirm button.
+    wrapper.find("TableDeleteConfirm button").last().simulate("click");
```

Comment:
Could we use a data-test here?

Reply:
You have the attr already on the element, `data-test="delete-discovery"`

Reply:
Thanks.